templates/vue/src/components/modals/common/FileUpload.vue

```diff
@@ -83,8 +83,8 @@
               required
               data-qa="import-file-input"
               @dragover.prevent
-              @drop.prevent="uploadFile"
-              @change="uploadFile"
+              @drop.prevent="handleFileUpload($event)"
+              @change="handleFileUpload($event)"
```

Comment:
I think this can be changed to simply
```suggestion
              @drop.prevent="handleFileUpload"
              @change="handleFileUpload"
```